coding-concepts.js

```diff
@@ -6,49 +6,106 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
+
+// a) Your answer: adds "indigo" to the end of the array.
 // b) Verify and explain:
 
+//      My guess:
+
+
+//      The variable colors that is equal to the array with all of the color strings incased in the brackets[].
+
+//      The .push() built-in method adds a value to the end of the array.
+
+//      The value in the array is the string "indigo".
+
+//      So i presume the code will display the the colors in the array with the addition of "indigo" at the end of the array.
+
+//      Incorrect.
+
+//      Correction:
+
+//      To get the answer i explained above, I'd have to add another console.log with the colors variable as the argument.
+
+//      the correct answer was 5 for the total number of strings in the projected code.
+
+//
+
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2023"
+// const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: 
+
+//      My guess:
+
+//      I say its going to be 10. The .length function call count in an Index format and it starts at 0 instead on 1. 
+//      There is 11 characters in the string so it will the character from 0 to 10. So the answer is 10.
```

Comment:
There are 10 characters in the string, not 11! Remember, length starts at 1, index is what starts at 0. In this example, the space counts as a character, which is why we get 10.